### PR TITLE
en_US/awesome/overview.md

### PR DESCRIPTION
**Pull request by CocosGames**

### CODE REVIEW

File: docs/en_US/config/overview.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Configuration File Format
+
+## Overview
+
+Configuration of V2Ray is a file with the following format，both for server and client，but the settings are different.
+
+If you are a beginner of V2Ray，please start from understanding Inbounds and Outbounds，filling in the necessary settings only and the application should work. Then learn about other components step-by-step，you will find that V2Ray is easy to use.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
If you are a beginner of V2Ray, please start from understanding Inbounds and Outbounds. Filling in the necessary settings only and the application should work. Then learn about other components step-by-step. You will find that V2Ray is easy to use.
```

File: docs/en_US/config/overview.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Configuration File Format
+
+## Overview
+
+Configuration of V2Ray is a file with the following format，both for server and client，but the settings are different.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
Configuration of V2Ray is a file with the following format, both for server and client, but the settings are different.
```

File: docs/en_US/config/overview.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    "outbounds": [],
+    "transport": {},
+    "stats": {},
+    "reverse": {}
+}
+```
+
+> `log`: [LogObject](#logobject)
+
+Log configuration，telling V2Ray how to print logs.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
Log configuration, instructing V2Ray how to print logs.
```

File: docs/en_US/config/overview.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+> `api`: [ApiObject](api.md)
+
+Remote control.
+
+> `dns`: [DnsObject](dns.md)
+
+Build-in DNS servers，if this file doesn't exist，use local DNS by default.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
Build-in DNS servers. If this field does not exist, use local DNS by default.
```

File: docs/en_US/config/overview.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+> `routing`: [RoutingObject](routing.md)
+
+Routing.
+
+> `policy`: [PolicyObject](policy.md)
+
+Local policy，permission-related configurations.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
Local policy, permission-related configurations.
```

File: docs/en_US/config/overview.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+> `inbounds`: \[ [InboundObject](inbounds.md) \]
+
+An array，each element is an inbound connection configuration.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
An array, each element of which is an inbound connection configuration.
```

File: docs/en_US/config/overview.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+> `outbounds`: \[ [OutboundObject](outbounds.md) \]
+
+An array，each element is an outbound connection configuration. The first element in the list is the primary outbound protocol. When the route matching cannot be found or the matching is not valid, the traffic is sent by the primary outbound protocol.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
An array, each element of which is an outbound connection configuration. The first element in the list is the primary outbound connection. When the route matching cannot be found or the matching is invalid, the traffic is routed to the primary outbound connection.
```

File: docs/en_US/config/overview.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+Statistics.
+
+> `reverse`: [ReverseObject](reverse.md)
+
+Reverse proxy.
+
+## LogObject
+
+`LogObject` is a json file which the configuration file uses in `log`.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
`LogObject` is a json field which the configuration file uses in `log`.
```

File: docs/en_US/config/overview.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+{
+    "access": "file path",
+    "error": "file path",
+    "loglevel": "warning"
+}
+```
+
+> `access`: string
+
+File path of log，it's a valid string path，such as `"/tmp/v2ray/_access.log"`（Linux）or`"C:\\Temp\\v2ray\\_access.log"`（Windows）. If it is empty，the log will be printed in stdout by default. V2Ray 4.20 added a special value `none`，means no access to log.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
File path of log. It should be a valid path string, such as `"/tmp/v2ray/_access.log"` (Linux) or`"C:\\Temp\\v2ray\\_access.log"` (Windows). If it is empty, the log will be printed to stdout by default. V2Ray 4.20 added a special value `none`, which means no access to log.
```

File: docs/en_US/config/overview.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+> `error`: string
+
+File path of error，it's a valid string path，such as`"/tmp/v2ray/_error.log"`（Linux）or`"C:\\Temp\\v2ray\\_error.log"`（Windows）. If it is empty，the error will be printed in stdout by default. V2Ray 4.20 added a special value `none`，means no access to error.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
File path of error log. It should be a valid path string, such as `"/tmp/v2ray/_error.log"` (Linux) or `"C:\\Temp\\v2ray\\_error.log"` (Windows). If it is empty, the error will be printed to stdout by default. V2Ray 4.20 added a special value `none`, which means no access to error.
```

Comment (DuckSoft) [reply docs/en_US/config/overview.md]: Suspection: Isn't the error log printed to stderr instead of stdout?

**Comment by CocosGames** [comment]
OK, it seems like a serious reading comprehension problem.

File: docs/en_US/config/overview.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+> `loglevel`: "debug" | "info" | "warning" | "error" | "none"
+
+Level of log. Default value is `"warning"`。
+
+* `"debug"`：Information for developers. All `"info"` included.
+* `"info"`：Running stats of V2Ray，no effect for the functions. All `"warning"` included.
+* `"warning"`：V2Ray encountered some problems, usually external problems, which do not affect V2Ray, but may affect the user experience. All `"error"` included.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
* `"warning"`：V2Ray may have encountered some problem, usually some external problem that does not affect V2Ray but possibly the user experience. All `"error"` included.
```

File: docs/en_US/config/overview.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+* `"error"`：V2Ray encountered a problem that needs to be resolved immediately.
+* `"none"`：nothing will be printed.
```
Comment (DuckSoft) [inline docs/en_US/config/overview.md]:
```suggestion
* `"none"`：Nothing will be printed.
```

**Review by DuckSoft — CHANGES_REQUESTED** [review]
Resolve and commit the changes to continue.

**Review by DuckSoft — APPROVED** [review]

**Pull request merged**